tests/kernel/lifo/lifo_usage/testcase.yaml

```diff
@@ -1,7 +1,8 @@
 tests:
   kernel.lifo.usage:
     # see #26646
-    platform_exclude: m2gl025_miv
+    # FIXME: See issue #39948
+    platform_exclude: m2gl025_miv qemu_cortex_a53_smp
```

Comment:
IIRC, `kernel.lifo.usage` fails intermittently on `qemu_cortex_a53_smp` because icount is not enabled for this platform.

Try re-running it with `-f` and it will eventually pass.

Reply:
worked for me on the 2nd try.